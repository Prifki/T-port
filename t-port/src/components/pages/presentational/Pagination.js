import React from 'react';

function Pagination(props) {
    return (
        <div className="pagination-wrapper">
            <div className="pagination">
                <a href="#" className="pagination--active">1</a>
                <a href="#">2</a>
            </div>
        </div>
    );
}

export default Pagination;
