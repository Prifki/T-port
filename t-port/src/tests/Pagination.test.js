import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Pagination from './../components/pages/presentational/Pagination';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Pagination', () => {
    it('should render correctly', () => {
      const output = shallow(
        <Pagination />
      );
      expect(shallowToJson(output)).toMatchSnapshot();
    });
});

