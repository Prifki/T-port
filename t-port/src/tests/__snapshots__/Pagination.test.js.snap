// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginMenu should render correctly 1`] = `
<div
  className="menu menu--login menu--login--active"
>
  <CloseButton />
  <h3
    className="menu__title"
  >
    Sign in
  </h3>
  <input
    placeholder="Login"
    type="text"
  />
  <input
    placeholder="Password"
    type="password"
  />
  <SubmitButton
    text="Sign in"
  />
</div>
`;

exports[`Pagination should render correctly 1`] = `
<div
  className="pagination-wrapper"
>
  <div
    className="pagination"
  />
</div>
`;
